digits/standard-networks/tensorflow/lenet_slim.py

```diff
@@ -0,0 +1,27 @@
+class UserModel(Tower):
```

Comment:
We have both standard and slim versions... but they are both buried under standard-networks. Should the slim be move somewhere else for more visibility? the default one will be the standard one?